iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -188,7 +188,9 @@ static bool isRootOp(Operation *op) {
     }
   }
   return isa<linalg::ConvInputNHWCFilterHWCFOp,
-             linalg::DepthwiseConvInputNHWCFilterHWCOp>(op);
+             linalg::DepthwiseConvInputNHWCFilterHWCOp,
```

Comment:
I think you also need to add these ops in two other places that has special checks for `ConvInputNHWCFilterHWCFOp` and alike.

Reply:
Thanks for the detailed explanation above, I added the check for pooling ops as well. I didn't do it because it is just working, and I wasn't sure if I should do it or not.